pkg_resources/__init__.py

```diff
@@ -65,40 +66,44 @@
 from os import open as os_open
 from os.path import isdir, split
 
-from pkg_resources.extern.jaraco.text import (
-    yield_lines,
-    drop_comment,
-    join_continuation,
-)
+if not TYPE_CHECKING:
```

Comment:
I used `not TYPE_CHECKING` to keep the diff easier to compare here. But it makes coverage report unhappy because it doesn't know that the `else` block should be skipped. If you don't mind, I can flip this condition.